ghconfig.py

```diff
@@ -36,6 +36,26 @@ def search_repositories(organisation, name):
     return repositories_found
 
 
+def set_branch_protection(organisation, repository, branch):
+    headers = {
+        'Authorization': 'Token {0}'.format(GH_TOKEN),
+        'Accept': 'application/vnd.github.loki-preview+json'
+    }
+
+    api_url = '{0}/repos/{1}/{2}/branches/{3}/protection'.format(
+        API_BASE_URL, organisation, repository, branch)
+
+    payload = {
+        "required_status_checks": None,
+        "required_pull_request_reviews": None,
+        "enforce_admins": True,
```

Comment:
Enforcing admins makes sure that the branch is restricted for admins too. Example: if you are admin you can't `git push --force origin master`. We can set it as `False` as default if you prefer.